index.bs

```diff
@@ -1269,11 +1269,11 @@ To <dfn for="SendStream">write</dfn> |chunk| to a {{SendStream}} |stream|, run t
 1. Let |bytes| be a copy of the [=byte sequence=] which |chunk| represents.
 1. Set |stream|'s [=[[PendingOperation]]=] to |promise|.
 1. Return |promise| and run the remaining steps [=in parallel=].
-1. Wait for |transport|'s [=[[Datagrams]]=]'s [=[[OutgoingDatagramsQueue]]=] to be empty.
-
-   Note: This means outgoing datagrams are prioritized over stream data.
 1. [=stream/Send=] |bytes| on |stream|'s [=[[InternalStream]]=] and wait for the operation to
    complete.
+   This sending MAY happen in parallel with sending of previously queued streams and datagrams,
```

Comment:
what does the term in parallel mean here? For a single connection, QUIC packets will still be emitted in some serial order. An implementation might timeslice (interleave) stream and datagram payload over different packets, but that might not align with some people's definition of parallel.